pallets/domains/README.md

```diff
@@ -10,51 +10,56 @@ The main storages of domains pallet are:
   Metadata associated with domain.
 - `DomainsByOwner: Map<AccountId, DomainName[]>`\
   Domains owned by account.
+- `DomainRecords: Map<DomainName, Map<RecordKey, RecordValueWithDeposit>>`\
+  All records associated with a given domain name.
 
 
-## Domain metadata
+## Domain Records
 
 In addition to resolving a domain owner, you can also resolve the various records set up
-by the user. The following table shows a list of records that can be attached to the domain name.
-
-
-<table>
-<thead>
-  <tr>
-    <th>Record Name</th>
-    <th>Type</th>
-  </tr>
-</thead>
-<tbody>
-  <tr>
-    <td rowspan="3">Content</td>
-    <td>String</td>
-  </tr>
-  <tr>
-    <td>IPFS</td>
-  </tr>
-  <tr>
-    <td>Hypercore</td>
-  </tr>
-  <tr>
-    <td rowspan="3">InnerValue</td>
-    <td>Subsocial Account</td>
-  </tr>
-  <tr>
-    <td>Subsocial Space</td>
-  </tr>
-  <tr>
-    <td>Subsocial Post</td>
-  </tr>
-  <tr>
-    <td>OuterValue</td>
-    <td>String</td>
-  </tr>
-</tbody>
-</table>
+by the user.
+
+Domains records system allows any dotsama domains to have key-value pairs of metadata attached to it.
+There is no restrictions on what value should be stored in what key, However we have a list of defined key schema to be 
+interpreted by clients.
+
+
+### Record Key standard
+
+Standard record key is split by namespaces by a . used as a separator.
```

Comment:
```suggestion
A standard record key is split by namespaces by a . used as a separator.
```

Reply:
I don't really understand what this means, can you give an example and I can try to rewrite this line more clearly?

Reply:
In the record system, anyone can just put jibberish as a key. example `key:"MyAwsomeName" => value:"Beef"`.

This flexibility is great but we want some sort of standardization so different clients from different teams can interpret and use these records in a meaningful way. That's why I suggest that we split the key into namespaces that are split by dot (`social.name => Tarek`)